test/QueryTest.cpp

```diff
@@ -8,6 +8,7 @@
 #include "addressbook.pb.h"
 #include "bank.pb.h"
 #include <infinispan/hotrod/ProtoStreamMarshaller.h>
+#include <infinispan/hotrod/JBasicMarshaller.h>
```

Comment:
@rigazilla Just for my information, what is the difference between the `JBasicMarshaller` and `BasicMarshaller`?

Reply:
oh the include is unnecessary. The BasicMarshaller just writes on the socket the object as it's stored in memory something like send(&object, sizeof(object)). The JBasicMarshaller implements a very small subset of the jboss marshaller features and it send objects as specified by the jboss River protocol. The result is that every object has a bytes preamble specifying type and size: i.e. for a small string is 0x03 0x3e 0x<size>.